src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -27,7 +27,8 @@ def probsolve_ivp(
     ivp,
     method="ekf0",
     which_prior="ibm1",
-    tol=None,
+    atol=None,
+    rtol=None,
     step=None,
```

Comment:
in principle, yes. Though I think this is more about the API of the ODE solvers (aka "how convenient should the convenience function be after all"), rather than the current steprules.

Reply:
Sure, I opened a new issue for it #272